doc/source/serve/production-guide/kubernetes.md

```diff
@@ -122,60 +122,62 @@ $ kubectl describe rayservice rayservice-sample
 ...
 Status:
   Active Service Status:
-    App Status:
-      Last Update Time:  2022-08-16T20:52:41Z
-      Status:            RUNNING
+    Application Statuses:
+      math_app:
+        Health Last Update Time:  2023-09-07T01:21:30Z
+        Last Update Time:         2023-09-07T01:21:30Z
+        Serve Deployment Statuses:
+          math_app_Adder:
+            Health Last Update Time:  2023-09-07T01:21:30Z
+            Last Update Time:         2023-09-07T01:21:30Z
+            Status:                   HEALTHY
+          math_app_DAGDriver:
```

Comment:
where's this coming from?

Reply:
These were from an old output, I forgot to remove